corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -326,48 +344,87 @@ def __init__(self, app_id, source_type, report_source_id, *args, **kwargs):
         self.helper.attrs['data_bind'] = "submit: submitHandler"
         self.helper.form_id = "report-config-form"
 
-        self.helper.layout = crispy.Layout(
-            self.top_fieldset,
-            FormActions(
-                crispy.ButtonHolder(
-                    crispy.Submit(
-                        'submit',
-                        _(self.button_text)
+        buttons = [crispy.Submit('submit', _(self.button_text))]
```

Comment:
just to clarify slightly, this would properly translate "Done" if there was a translation for "Done" in the .mo/.po files. What it won't do is add that string automatically to those files (so it would have to show up elsewhere or be added manually).

The `ugettext_lazy` suggestion is still the right way to handle it (or `ugettext_noop` + keeping this line in)